docs/apis-clients/java-client/index.md

```diff
@@ -86,6 +86,10 @@ ZeebeClient client =
         .build();
 ```
 
+## Javadoc
+
+The official Java client library API documentation can be found [here](https://javadoc.io/doc/io.camunda/zeebe-client-java). These are standard Javadocs, so your favorite JVM IDE will be able to install them locally as well.
```

Comment:
💭 When we archive a previous version of our docs this link is not adjusted. This could be an argument to ignore my suggestion in the versioned docs. Alternatively, we could adjust the archiving process to put the version number in that URL.

Reply:
@akeller What are your thoughts here? I'd like to ensure maintenance is as low as possible but interested in your opinion.

Reply:
Link to the versionless javadocs, please. We can revisit this if we get feedback or have other things we change in the release process.

FYI @pepopowitz might be a good future topic if we have a collection of these items as we move from one version to the next.